cif/products/src/mappers/PagedResponseProductMapper.ts

```diff
@@ -112,4 +118,25 @@ export default class PagedResponseProductMapper extends Mapper<PagedResponseProd
       totalResults,
     });
   }
+
+  private buildCIFRequest(mappable: any) {
+
+    const {
+      offset = 0,
+      limit = 20,
+      filter = '',
+    } = mappable;
+
+    const cifRequestType = CommerceIntegrationHelper.checkRequestType(filter);
+    switch (cifRequestType) {
+      case CIFRequestType.Variant: {
+        const selectedVariant = filter.match(/(?<=")[^"]+(?=")/);
```

Comment:
This is the same line as 137 why duplicate code inside the cases?

Reply:
ok. Now created another method in helper which extracts id from the query filter. have a look.